neo/public/javascripts/map/controllers.js

```diff
@@ -1,6 +1,9 @@
 angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
   .controller('MapCtrl', function($scope, $window, $http, $compile, Asterix, leafletData) {
     $scope.result = {};
+    $scope.doNormalization = false;
+    $scope.upscaleFactor = 1000 * 1000;
+    $scope.upscaleFactorText = "M";
```

Comment:
I think it should be in js, since it's front-end related. Do we have some global config file or object for the TwitterMap right now? If the case, we can add these two options to it. If not, maybe we can have one...